flink-runtime/src/main/java/org/apache/flink/runtime/util/config/memory/jobmanager/JobManagerFlinkMemoryUtils.java

```diff
@@ -91,13 +91,13 @@ private static void verifyJvmHeapSize(MemorySize jvmHeapSize) {
 	private static void verifyJobStoreCacheSize(Configuration config, MemorySize jvmHeapSize) {
 		MemorySize jobStoreCacheHeapSize =
 			MemorySize.parse(config.getLong(JobManagerOptions.JOB_STORE_CACHE_SIZE) + "b");
-		if (jvmHeapSize.compareTo(jobStoreCacheHeapSize) < 1) {
+		if (jvmHeapSize.compareTo(jobStoreCacheHeapSize) < 0) {
```

Comment:
same here, why zero?

Reply:
@azagrebin The same reason mentioned above.